app/views/admin/people/_form.html.erb

```diff
@@ -3,38 +3,118 @@
   show_consult_gds_warning ||= false
 %>
 
-<%= form_for [:admin, person] do |person_form| %>
-  <%= person_form.errors %>
-
-  <fieldset>
-    <%= person_form.check_box :privy_counsellor, label_text: 'Rt Hon?' %>
-    <%= person_form.text_field :title %>
-    <%= person_form.text_field :forename %>
-    <%= person_form.text_field :surname %>
-    <%= person_form.text_field :letters %>
-  </fieldset>
-  <fieldset class="image">
-    <%= person_form.label :image, 'Image' %>
+<%= form_for [:admin, person] do |form| %>
+  <%= hidden_field_tag 'person[privy_counsellor]', "0" %>
+
+  <%= render "govuk_publishing_components/components/checkboxes", {
+    id: "person_privy_counsellor",
+    name: "person[privy_counsellor]",
+    items: [
+      {
+        label: "Rt hon?",
+        value: "1",
+        checked: person.privy_counsellor
+      }
+    ]
+  } %>
+
+  <%= render "govuk_publishing_components/components/input", {
+    label: {
+      text: "Title",
+      heading_size: "l",
+    },
+    name: "person[title]",
+    id: "person_title",
+    value: person.title,
+    error_items: errors_for(person.errors, :title)
+  } %>
+
+  <%= render "govuk_publishing_components/components/input", {
+    label: {
+      text: "Forename",
+      heading_size: "l",
+    },
+    name: "person[forename]",
+    id: "person_forename",
+    value: person.forename,
+    error_items: errors_for(person.errors, :forename)
+  } %>
+
+  <%= render "govuk_publishing_components/components/input", {
+    label: {
+      text: "Surname",
+      heading_size: "l",
+    },
+    name: "person[surname]",
+    id: "person_surname",
+    value: person.surname,
+    error_items: errors_for(person.errors, :surname)
+  } %>
+
+  <%= render "govuk_publishing_components/components/input", {
+    label: {
+      text: "Letters",
+      heading_size: "l",
+    },
+    name: "person[letters]",
+    id: "person_letters",
+    value: person.letters,
+    error_items: errors_for(person.errors, :letters)
+  } %>
+
+  <%= render "govuk_publishing_components/components/fieldset", {
+    legend_text: "Image",
+    heading_size: "l"
+  } do %>
     <% if person.image_url %>
-      <%= image_tag person.image_url %>
+      <%= image_tag person.image.url(:s465) %>
```

Comment:
What does `:s465` do?

Reply:
I'm guessing related to file size.

Maybe what we need to make this work for all screens is to add a wrapper around it to stop the image from overflowing.

Reply:
Is the best way to do this just to do something like

```
responsive-image { 
  max-width: 100%; 
  height: auto;
}
```

in the stylesheets and add the wrapper?

Where should the styling live. It won't be view specific. Should we have a top level file? How do we make it BEM compliant?

Reply:
Maybe something like this - I would put it in a separate view CSS file
```scss
.app-view-edit-person__image { 
  max-width: 100%; 
  height: auto;
}
```

Reply:
Isn't this something we will want to reuse on things like the show page (and anywhere else in Whitehall we use images?)

Reply:
I tend to agree with @davidgisbey – this does seem like a useful utility class that'd end up being used in more than one place. But I can't think of any examples off the top of my head – so maybe something to do later if/when the need arises. (I suspect other MoG pages might end up needing it.)